### ISSUE TITLE
The egg ticket updater isn't working, neither is the upload local slot1.json function

### ISSUE THREAD

**Issue by nyanzan** [fire6945/RogueEditor#4]
Exactly what the title says

**Comment by fire6945** [comment]
Can you elaborate more on what happened?

**Comment by nyanzan** [comment]
Nothing happened. I tried them both out and it didn't work.

**Comment by FuecocoBoi09** [comment]
IIRC it sets your tickets not adds tickets

**Comment by nyanzan** [comment]
Yeah that's what I meant lol. I just tried both again and curiously it does seem to be working now even though I did the exact same thing lol. Maybe it was a server issue? Thank you for responding. If I could make a suggestion to the program, would it be possible to add a function that could add specific items to your current save slot like the Map, Shiny Charm, Ability Charm or Soothe bell so that you don't need to add them manually to every file? Thank you for your time.
